### DISCUSSION TITLE
IPCC reports

### DISCUSSION THREAD

**Discussion by mrchristian** [singlesourcepub/community#43] [Ideas]
Interesting thread on making the IPCC reports single source. IPCC have made a position available to deal with making an HTML output from their Indesign workflow. But the wider community is getting involved as its a slow process and it would appear that a community processing of PDFs would enable the same results.

See the thread: https://twitter.com/themitak/status/1511540675094884355

SSPC should get involved eh? We could pool our efftors and help!

Shall we get a call and see who's interested and then offer help?

**Comment by tarleb** [comment]
I'd be in!

**Comment by mrchristian** [comment]
There will be a closed workshop on the 24th Sept for Sematic Enrichment of IPCC reports - https://github.com/petermr/semanticClimate - I'll share details at the community call or here if people are interested?

**Comment by tarleb** [comment]
Can you post it here, or maybe message me directly? I'd like to attend, but I'm not sure yet if I'll find the time.

**Comment by mrchristian** [comment]
I'm waiting to hear back from the organisers about places at the workshop / hackathon. Numers are only restricted as it is their first hackathon on the IPCC publications. 

The hackathon will be to run TDM and ML on HTML versions of chapters of https://www.ipcc.ch/report/ar6/wg3/downloads/report/IPCC_AR6_WGIII_Full_Report.pdf and semantify the content with WikiData terms.

Here is some info from the organisers:

QUOTE

Delhi EffecDelhi Effective Education & Pedagogy Cluster (DEEP-C) 

DEEP-C is one of the verticals of Delhi City Knowledge Cluster, managed by [Delhi Research Implementation and Innovation](https://www.driiv.co.in/). This City Cluster (and [four others across India](https://www.psa.gov.in/st-clusters)) were established by the Office of the Principal Scientific Advisor (PSA) to the Government of India. 

Under the DEEP-C/DRIIV banner, and in the spirit of Unlocking Knowledge for Open Science we are organising a half day online Climate Knowledge Hunt Hackathon, and I take this opportunity to invite you to join us for this Hackathon from 3PM - 6:30 PM IST (== 10:30AM - 2PM BST) on 24th September 2022. 

We are hoping to be joined by like minded Climate Enthusiasts like yourself and others at the [ClimateCollective](https://climatecollective.net/),  [WikiData](https://www.wikidata.org/wiki/Wikidata:Main_Page), [Shuttleworth](https://shuttleworthfoundation.org/), [WikiMedia](https://www.wikimedia.org/), [WhoseKnoweldge](https://whoseknowledge.org/) and [University of Geneva](http://cui.unige.ch/~grey/)

The first hour of the event will be the Inaugural Session, and you will be glad to know that Dr. P Maini, Scientific Secretary at the Office of the PSA to GoI, (who is a Woman in Climate Change herself!) has kindly agreed to deliver the Keynote address at this session! Apart from our Collaborators, We wish to bring 4-5 Climate Change organisations (Max 45) on board to retain cohesiveness and more interaction at the event.

Aim of the Climate Knowledge Hunt Hackathon:
1.	Understanding and creating climate knowledge (esp. IPCC)
2.	Learning/Developing human/machine knowledge systems
3.	Making new contacts in the India Climate Landscape
4.	Feedback knowledge and technology to ClimateHunt

More Info:
•	We believe in Learning By Doing and Making Knowledge Freely accessible to all. 
•	This Hackathon is planned as a half day invited-only event to showcase our open source policy and research/solutions.
•	After the Inaugural session, we will begin the collaborative Hackathon and it will involve teaching Climate Enthusiasts how to extract knowledge from locked literature (such as IPCC reports in PDF format!) - using our in-house (Open source!) tools and materials over Google Collab and Jupyter notebooks. 
•	No prior coding skills required - we are handling the documentation so that our Climate Knowledge Hunt is automated yet comprehensible !
•	All our materials and solutions are developed and will be delivered by young Indian (including rural) Intern students as part of  Education & Pedagogy Cluster (DEEP-C) 

END QUOTE

**Reply by mrchristian** [reply]
The organisers just got back. Yes I can invite people. Currently I would only have two open slots for SSPC, but there will be more hacks.

@tarleb let me know if you want to come along. Sat, September 24, 2022

3:00 PM – 6:30 PM IST

[Ah, the organisers just posted a web link.](https://www.eventbrite.co.uk/e/the-climate-knowledge-hunt-hackathon-tickets-414825362827)
![DeepC_Climate_Hackathon_Flyer_Sep22_sml](https://user-images.githubusercontent.com/772579/189090042-b625bc1d-7de8-4cf5-97c4-8556aff6e18f.jpg)

But I also think SSPC with friend should also organise a hack during OA week @isabellameinecke - we could get a few groups together and make a chapter into an interoperable format, and do some semantification :-)

**Comment by mrchristian** [comment]
Hi,

Here is a reminder invite for the Climate Hack hackathon. Saturday 24 Sept, 12.00-15:00 CEST.

Invite to a Climate Hackathon this Saturday, September 24, 2022, 3:00 PM – 6:30 PM IST (12:00-15:00 CEST).

I have five slots left for the session – which does not require any prior knowledge of technology or reports. Please RSVP below and let me know if you can attend.

THE CLIMATE KNOWLEDGE HUNT HACKATHON
Collaborative Hackathon for OpenScience: Shared Learning with Climate Enthusiasts to extract knowledge from locked literature (PDF reports)

https://www.eventbrite.co.uk/e/the-climate-knowledge-hunt-hackathon-tickets-414825362827

In a nutshell it will be an inaugural hackathon to bring together groups on liberating IPPC reports, and specifically ‘Sixth Assessment Report’ the IPCC most up-to-date climate change assessment. The report is publishing as a 3,000 page PDF. The hackathon goals are to transform the report into an interoperable format, semantify it with Wikidata and a variety of other methods, create a human/machine enrichment and translation workflows, as well as facilitate engagement, reuse, and citizen science.

Existing work: https://github.com/petermr/semanticClimate 

About the report:

Quote: ‘The IPCC has finalized the third part of the Sixth Assessment Report, Climate Change 2022: Mitigation of Climate Change, the Working Group III contribution. It was finalized on 4 April during the 14th Session of Working Group III and 56th Session of the IPCC’.

Overview
https://www.ipcc.ch/report/sixth-assessment-report-working-group-3/

Full 3000 page PDF
https://www.ipcc.ch/report/ar6/wg3/downloads/report/IPCC_AR6_WGIII_Full_Report.pdf

Policy maker summary
https://www.ipcc.ch/report/ar6/wg3/downloads/report/IPCC_AR6_WGIII_SPM.pdf

**Reply by tarleb** [reply]
Thank you Simon! I'm quite disappointed that I'll have to pass this opportunity, as it's at the same time as my daughter's birthday party. :slightly_frowning_face:

**Reply by mrchristian** [reply]
Oh have a great party :-) Fun. No problem, there will be plenty more hackathons - this is only the start. Have a great day!

**Comment by mrchristian** [comment]
From the Semantic Climate hack a nice task dropped out of the session. How can we improve the PDF to HTML conversion - I think this question falls perfectly into SSPCs hands and we can run with this one. Please see the details here in this GitHub issue - https://github.com/petermr/semanticClimate/issues/10

**Comment by tarleb** [comment]
I'd be in!

**Comment by xldrkp** [comment]
Just for the record: semanticClimate was [invited to a webinar of UN Library](https://research.un.org/conferences/webinar3) to give insights into the activities to liberate climate-related scholarly literature.
